roles/docker/tasks/main.yml

```diff
@@ -36,3 +36,39 @@
   when:
     - l_use_crio
     - openshift_docker_is_node_or_master | bool
+
+- when:
+  - l_use_crio
+  - l_use_system_container
+  block:
+  - name: stop the current running docker
+    systemd:
+      state: stopped
+      name: docker
```

Comment:
I believe this should be ``"{{ openshift.docker.service_name }}"`` ... though I _think_ at this point neither ``docker`` nor ``container-engine`` can be assumed to be pre-installed.